lib/ruby-stackoverflow/client/user_helper.rb

```diff
@@ -147,8 +147,8 @@ def user_full_reputation_history(id, options = {})
 
       private
 
-      def user_response(options={}, url='')
-        url =  'users/'+ url
+      def user_response(options = {}, url = '')
+        url =  'users/' + url
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.